tooling/template-base/components/Navigation/UserLoginNavBar.tsx

```diff
@@ -14,7 +14,7 @@ export default function UserLoginNavBar() {
           <div className="flex items-center justify-between md:block">
             {/* logo */}
             <div className="">
-              <Logo />
+              <Logo logoLink="" style="" />
```

Comment:
Not sure where the additional props came from? Don't believe the `Logo` component makes any use of those.

Reply:
Hey, the Logo component is actually supposed to have props as the styling may change such as size based on use case. But I think I should remove the linkUrl as it will always redirect to homepage unless you can think of another use case?

Reply:
that's a good approach, will implement now